test/browser/callbackify.js

```diff
@@ -167,3 +167,13 @@ test('util.callbackify non-function inputs throw', function (t) {
   });
   t.end();
 });
+
+test('util.callbackify resulting function should have one more argument', function (t) {
+  // Test that resulting function should have one more argument
+  [async () => { }, async (a) => { }, async (a, b) => { }].forEach(function (fct) {
```

Comment:
we can't use async function syntax, or arrow function syntax, in tests that need to run down to node 0.6.

Reply:
I can remove the async keyword here, sure ! but ...

- There are other places in the tests where  `async` keyword is already used.

![image](https://user-images.githubusercontent.com/680220/169121300-150eb1f4-3052-4da1-9fb1-2976e6838565.png)

-  .travis.yml shows that tests are only run with NodeJS > 10 ,  it looks like NodeJS <10 version are now ignored by node--util.

Reply:
oof, yes, that's sadly the case :-/ don't worry about removing them for now then

Reply:
callbackify-async is a test that only runs in node.js versions that *do* support async functions. the other test files should not use them.